src/sentry/conf/server.py

```diff
@@ -2138,6 +2138,8 @@ def create_partitioned_queues(name):
 # Where to write the lockfile to.
 MIGRATIONS_LOCKFILE_PATH = os.path.join(PROJECT_ROOT, os.path.pardir, os.path.pardir)
 
+ZERO_DOWNTIME_MIGRATIONS_RAISE_FOR_UNSAFE = True
```

Comment:
What's this setting for?

Reply:
When this is `False` it will just print warnings about the migration being unsafe, vs raising an exception and failing

Reply:
Got it. 👍